sport/app/AppLoader.scala

```diff
@@ -31,6 +32,9 @@ class AppLoader extends FrontendApplicationLoader {
 
 trait SportServices {
   def wsClient: WSClient
+  lazy val footballClient = wire[FootballClient]
+  lazy val competitionDefinitions = CompetitionsProvider.allCompetitions
+  lazy val competitionsService = wire[CompetitionsService]
```

Comment:
I think it makes sense to have it defined like that. That's indeed a limitation (feature?) of macwire.